fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/utils/RelaxedPropertiesHelper.java

```diff
@@ -147,8 +147,9 @@ private static Calendar parseExpectedXsdDateTimeValue(final RDFNode node) {
      * @param triple the triple to check.
      */
     public static void checkTripleForDisallowed(final Triple triple) {
-        if (triple.getPredicate().equals(type().asNode()) && !triple.getObject().isURI()) {
-            // The object of a rdf:type triple is not a URI.
+        if (triple.getPredicate().equals(type().asNode()) && !triple.getObject().isVariable() &&
```

Comment:
Hadn't considered variables being supplied for RDF types... Is this for PATCH requests that set types based on a variable? Are there any tests that demonstrate this restriction?

Reply:
Only PATCH requests seem to have variables but this was needed based on the test I wrote which had something like
```
DELETE { <> rdf:type ?type } INSERT { <> rdf:type <some rdf type> } WHERE { <> rdf:type ?type }
```
In that case this method gets a Variable as the object of the triple. So I didn't want to throw an error for it.